ansible/README.md

```diff
@@ -0,0 +1,16 @@
+# Ansible Roles
+=========
+
+## Structure
+- **ansible**
+This is where the various roles resides.
+
+## Prerequisites
+- Ansible
+- Python
+
+## Role List
+### 1. User-management 
+[User management in various tools](/ansible/user-management/main.yml)
+
```

Comment:
This role helps you to manage/creates users in different platforms like AWS, Linux, Jenkins, MySQL and Postgres.

Reply:
This is updated.